### PR TITLE
Ben hft pipeline

### PR DESCRIPTION
**Pull request by benfoley**
Replace the transcriber code with HFT pipeline, which can handle long audio files. 

Also fix a niggling issue with GUI not handling i18n for the tier selection mechanism.

### CODE REVIEW

**Comment by benfoley** [comment]
I'll make a seperate PR with GUI for transcribing using a model on the hft hub.

**Comment by benfoley** [comment]
transcribing

File: elpis/engines/hft/objects/transcription.py [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
 from elpis.engines.hft.objects.model import HFTModel
 
+LOAD_MODEL = "load_model"
 LOAD_AUDIO = "load_audio"
-PROCESS_INPUT = "process_input"
 TRANSCRIPTION = "transcription"
 SAVING = "saving"
 
-STAGES = [LOAD_AUDIO, PROCESS_INPUT, TRANSCRIPTION, SAVING]
+STAGES = [LOAD_MODEL, LOAD_AUDIO, TRANSCRIPTION, SAVING]
```
Comment (nicklambourne) [inline elpis/engines/hft/objects/transcription.py:23]: This should be an enum.

File: elpis/engines/hft/objects/transcription.py [hunk 2]
```diff
@@ -48,50 +48,38 @@ def __init__(self, **kwargs) -> None:
...
-        with torch.no_grad():
-            logits = model(input_values).logits
-        predicted_ids = torch.argmax(logits, dim=-1)
-        self._set_stage(PROCESS_INPUT, complete=True)
-
-        # Transcribe
         self._set_stage(TRANSCRIPTION)
-        transcription = processor.decode(predicted_ids[0])
-        logger.info(f"{transcription=}")
+        logger.info("=== Inference pipeline")
```
Comment (nicklambourne) [inline elpis/engines/hft/objects/transcription.py:65]:
```suggestion
        logger.info("=== Inference pipeline ===")
```

File: elpis/engines/hft/objects/transcription.py [hunk 3]
```diff
@@ -193,15 +122,17 @@ def _save_utterances(self, utterances) -> None:
...
         result = pympi.Elan.Eaf(author="elpis")
-
         result.add_linked_file(self.audio_filename)
         result.add_tier("default")
-
         to_millis = lambda seconds: int(seconds * 1000)
-        for word, start, end in zip(*utterances):
-            start, end = to_millis(start), to_millis(end) + 1
+        for utterance in utterances:
+            print(utterance)
```
Comment (nicklambourne) [inline elpis/engines/hft/objects/transcription.py:129]: I suspect this line can be removed, but if not:
```suggestion
            logger.info(utterance)
```

**Review by nicklambourne — CHANGES_REQUESTED** [review]

File: elpis/engines/hft/objects/transcription.py (hunk 1 above)
Comment (benfoley) [reply elpis/engines/hft/objects/transcription.py]: changed this and TRANSCRIPTION_STATUS to enums in next commit

File: elpis/engines/hft/objects/transcription.py (hunk 3 above)
Comment (benfoley) [reply elpis/engines/hft/objects/transcription.py]: removed

File: elpis/engines/hft/objects/transcription.py (hunk 2 above)
Comment (benfoley) [reply elpis/engines/hft/objects/transcription.py]: changed this and others that were ==== to use ===

**Review by nicklambourne — APPROVED** [review]

**Pull request merged**